### DISCUSSION TITLE
509: certificate is valid for api-int.lab.<mydomain>, not api.lab.<mydomain>

### DISCUSSION THREAD

**Discussion by ghost** [okd-project/okd#938] [Q&A]
**Describe the bug**

**Version**
<!-- Please specify OKD version release: 4.8.0-0.okd-2021-10-10-030117
Its a UPI installation

**How reproducible**
<!-- Please specify how often the issue is reproducible - 100%,

**Log bundle**
ERROR Bootstrap failed to complete: Get "https://api.lab.<mydomain.com>:6443/version?timeout=32s": x509: certificate is valid for api-int.lab.<mydomain.com>, not api.lab.<mydomain.com>
ERROR Failed waiting for Kubernetes API. This error usually happens when there is a problem on the bootstrap host that prevents creating a temporary control plane. 
FATAL Bootstrap failed to complete           

For me its the first time encountering this issue relating certificates when installing a freshly new Cluster with 3 control planes and 3 worker nodes.

**Comment by vrutkovs** [comment]
Please attach [log bundle](https://docs.okd.io/latest/installing/installing-troubleshooting.html#installation-bootstrap-gather_installing-troubleshooting)
